index.html

```diff
@@ -6,8 +6,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Greenville OpenMapData</title>
   </head>
-  <body class="bg-slate-900 text-slate-300 h-screen w-screen m-0 p-0">
-    <div id="app" class="h-screen w-screen" />
+  <body class="bg-slate-900 text-slate-300 h-full-screen w-screen m-0 p-0">
```

Comment:
:pick: I'm not really seeing any benefit of using `h-full-screen` over just adding `fixed h-full overscroll-none` inline, but I guess it does give a hint of why these three styles might be special

Reply:
@oliviasculley I did kind of want to separate that combo out just in case further issue rose up to help make it easier to find which combo is the one that should be altered. I'm not sure how I feel about using a class name that looks so similar to a Tailwind class., though.. Do you think it would be less confusing to give it a more distinct name?

Reply:
I think it just seemed like a bit of a smell but not really even in the code itself. Maybe we're looking at this wrong, and it might be more ideal to build a cypress test that can somehow ensure that the navbar is visible on mobile views so we don't have to worry about any particular solution. However, given that we might move away from the tabbed/navbar navigation to replacing the About page with a modal, keeping this name is probably a good enough solution in the meantime until we know what we want to assert on better in the future